src/main/java/tasks/Todo.java

```diff
@@ -0,0 +1,44 @@
+package tasks;
+
+import duke.DukeException;
+
+public class Todo extends Task {
```

Comment:
This is minor point - perhaps you could consider renaming the class `Todo` to `ToDo` (Pascal Case) to follow the coding standards?

Reply:
Oh my - I completely missed this out. 😅 Thank you for pointing it out!